src/SkillService.Data/SkillRepository.cs

```diff
@@ -13,6 +14,32 @@ public class SkillRepository : ISkillRepository
   {
     private readonly IDataProvider _provider;
 
+    private IQueryable<DbSkill> CreateFindPredicates(
+      FindSkillFilter filter,
+      IQueryable<DbSkill> dbSkill)
+    {
+      if (!string.IsNullOrEmpty(filter.NameIncludeSubstring))
+      {
+        dbSkill = dbSkill.Where(
+          skill =>
+            skill.Name.Contains(filter.NameIncludeSubstring));
+      }
+
+      if (filter.IsAscendingSort.HasValue)
+      {
+        dbSkill = filter.IsAscendingSort.Value
+          ? dbSkill.OrderBy(skill => skill.Name)
+          : dbSkill.OrderByDescending(skill => skill.Name);
+      }
+
```

Comment:
remove

Reply:
It seems like it'll make impossible to perform a sorting by an ascending/descending order. Should still I remove this?

Reply:
i think it means "remove empty line"))